modules/ROOT/pages/to-request-access.adoc

```diff
@@ -22,6 +22,10 @@ applications.
 API providers can view, manage, and revoke these contracts by following the steps in
 xref:api-manager::manage-client-apps-latest-task.adoc[Managing API Instance Contracts on API Manager].
 
+A client application's owners have permission to request access for that client application.
+To request access for a client application created by another user, add your user account to
```

Comment:
> add your user account to the application owners list in Anypoint API Manager
users don't have the possibility of adding themselves to the owners list, I think we should make it clear that they need to ask the owner of the client application to add them

Reply:
Good point! Thank you, Mariano. Fixed. Would you please review the new version?